### PR TITLE
Added Add Jitter check box to the Describe > Graphs >Scatterplot dialog

### LINKED ISSUE #8524: Add jitter to the Describe > Graphs >Scatterplot dialog
@N-thony this should be a simple improvement to allocate.

Here is the current dialog with the ggplot2 diamonds data.

image

a) Could you please add a checkbox with label Jitter.  Default is unchecked.  If checked then it replaces geom_point, see below, by geom_jitter.  
b) There are also 2 up/down controls.  The first is lebelled Width and the second is Height.  The up-down has default 0.4 in each case.  It goes from 0.00 to 0.5 in steps of 0.01.
c) Add Jitter to the Plot options pull-down.

Here is the code from that dialog:

`last_graph <- ggplot2::ggplot(data=diamonds, mapping=ggplot2::aes(x=color, y=cut)) + ggplot2::geom_point() + theme_grey()`

If you just change point to jitter in the code it changes the plot.

You may wish to incude the width amd height parameters of jitter always, or just when they are different to 0.4, which is the R default.

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes #8524
@rdstern, @N-thony I Added Jitter check box on ScatterPlot Dialog. Please have a look.

### CODE REVIEW

**Review by rdstern — DISMISSED** [review]
@MeSophie that looks great.  @lloyddewit over to you.

File: instat/dlgScatterPlot.vb [hunk 1]
```diff
@@ -129,6 +136,18 @@ Public Class dlgScatterPlot
...
 
+        ucrNudHeigth.SetParameter(New RParameter("height", 5))
+        ucrNudHeigth.Maximum = 0.5
+        ucrNudHeigth.Minimum = 0.00
+        ucrNudHeigth.Increment = 0.01
+        ucrNudHeigth.DecimalPlaces = 2
+
+        ucrNudWidth.SetParameter(New RParameter("width", 6))
+        ucrNudWidth.Maximum = 0.5
+        ucrNudWidth.Minimum = 0.00
```
Comment (lloyddewit) [inline instat/dlgScatterPlot.vb:147]:
```suggestion
        ucrNudHeigth.Minimum = 0
        ucrNudHeigth.Increment = 0.01
        ucrNudHeigth.DecimalPlaces = 2

        ucrNudWidth.SetParameter(New RParameter("width", 6))
        ucrNudWidth.Maximum = 0.5
        ucrNudWidth.Minimum = 0
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@MeSophie Looks, good, just a couple of very small suggestions

**Review by rdstern — APPROVED** [review]
@lloyddewit this still looks fine to me

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**